src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1682,17 +1692,15 @@ private void SynchronizeInternal(GraphSyncData syncData)
                 // Get AST list that need to be executed
                 var finalDeltaAstList = changeSetComputer.GetDeltaASTList(syncData);
 
+                if (!DSExecutionEngine)
+                {
+                    CompileAndExecuteMSIL(finalDeltaAstList);
+                }
+
                 // Prior to execution, apply state modifications to the VM given the delta AST's
                 bool anyForcedExecutedNodes = changeSetComputer.csData.ForceExecuteASTList.Any();
                 changeSetApplier.Apply(runnerCore, runtimeCore, changeSetComputer.csData);
 
-                ///////////////////////////////////////////////////
-                Dictionary<string, IList> input = new Dictionary<string, IList>();
-                var codeGenIL = new EmitMSIL.CodeGenIL(input, @"D:\GitHub\Dynamo\src\Engine\EmitMSIL\bin\Debug\opCodes.txt");
-                codeGenIL.Emit(finalDeltaAstList);
-
-                ///////////////////////////////////////////////////
-
                 if (finalDeltaAstList.Any() || anyForcedExecutedNodes || changeSetComputer.csData.ModifiedNodesForRuntimeSetValue.Any())
```

Comment:
This is still confusing to me.  So when DSExecutionEngine is true, this condition will in most cases be false? At least now in the beginning?